modules/gitops-release-notes-1-14-0.adoc

```diff
@@ -0,0 +1,106 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes-1-14-0.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-14-0_{context}"]
+= Release notes for {gitops-title} 1.14.0
+
+{gitops-title} 1.14.0 is now available on {OCP} 4.12, 4.13, 4.14, 4.15, and 4.16.
+
+[id="errata-updates-1-14.0_{context}"]
+== Errata updates
+
+[id="RHSA-XXXX:NNNN-gitops-1-14-0-security-update-advisory_{context}"]
+=== RHSA-XXXX:NNNN - {gitops-title} 1.14.0 security update advisory
+
+Issued: 2024-09-18
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-XXXX:NNNN[RHSA-XXXX:NNNN]
+
+If you have installed the {gitops-title} Operator in the default namespace, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-gitops-operator
+----
+
+[id="new-features-1-14-0_{context}"]
+== New features
+
+* With this update, users can customize permissions for the Application Controller component in a cluster-scoped Argo CD instance. The {gitops-title} Operator establishes an aggregated `ClusterRole`, and users can configure its permissions by following the standards of an aggregated `ClusterRole`. link:https://issues.redhat.com/browse/GITOPS-4681[GITOPS-4681]
+
+* With this update, the following new features have been introduced for multi-sourced application:
+** Users can access revision history and rollback to a specific version of a multi-source application. Specifically, the *Details* view's panel displays the most recent revision or commit details for each source in the multi-source application. Also, the *History* and *Rollback* panel displays the revision history associated with each source, enabling users to select the desired version for rollback. link:https://issues.redhat.com/browse/GITOPS-4647[GITOPS-4647]
```

Comment:
Not sure what this actually looks like, but "*Details* views panel" reads kind of funny. Can you ensure that's correct? Could it be *Details* -> *View*  panel? or *Details view* panel?

Reply:
I am keeping it as Details View panel. Thanks.